docs/UserGuide.md

```diff
@@ -348,9 +352,24 @@ _Module prefix refers to the first two characters of every module name._
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install Mass Linkers in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Mass Linkers home folder.
 
+**Q**: Do I need internet connection to use Mass Linkers?<br>
+**A**: No, internet connection is not required. 
+
+**Q**: I am using Mac and I tried opening the `MassLinkers.jar` file by double-clicking it in Finder. It says *"MassLinkers.jar" cannot be opened because it is from an unidentified developer.*<br>
+**A**: Right click the `MassLinkers.jar` file and select `Open with JavaLauncher(default)`.
+
+**Q**: Will the data of batchmates be saved if the program was not closed via the ```exit``` command?<br>
+**A**: Yes, the data will still be saved. However, we advise using the ```exit``` command for a better user experience.
```

Comment:
Maybe we can be more explicit by saying something like this:

Yes, the data will still be saved if the program is closed by closing the application window directly...